File: api/Domain/Models/Heroi.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Models
{
    public class Heroi
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string NomeHeroi { get; set; }
        public DateTime DataNascimento { get; set; }
        public float Altura { get; set; }
        public float Peso { get; set; }
    }
}

File: frontend/src/app/models/heroi.ts
import { Superpoderes } from "./superpoderes";

export class Heroi{
  id?: number;
  nome: string;
  nomeHeroi: string;
  dataNascimento: Date;
  Altura: number;
  Peso: number;
  Superpoderes: Array<Superpoderes>
}
